### DISCUSSION TITLE
AppControl Manager: What is the best way to add a new 'deny' entry to a base policy?

### DISCUSSION THREAD

**Discussion by bladehstream** [HotCakeX/Harden-Windows-Security#439] [General]
It is easy to allow additional things with supplemental policies, however there doesn't seem to be a way in AppControl manager to add a binary/other file to the deny list. Is this a roadmap item? Would prefer not to have to go into the xml and edit it manually :D

**Comment by HotCakeX** [comment]
Yes it's coming soon!

**Reply by bladehstream** [reply]
Hooray! Thank you very much!

**Reply by HotCakeX** [reply]
You're welcome!

**Comment by HotCakeX** [comment]
Hi, you can create Deny policies using AppControl Manager now
https://github.com/HotCakeX/Harden-Windows-Security/releases/tag/AppControlManager.v.1.7.0.0
